four_wheel_steering_controller/include/four_wheel_steering_controller/speed_limiter.h

```diff
@@ -0,0 +1,137 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2013, PAL Robotics, S.L.
```

Comment:
this is a lot of copied code from the `diff_drive_controller`, could you use that one instead?

Reply:
Maybe better not...I'm a bit hesitant on this: methodically that's the right choice but from a release and maintenance point of view it's better if things are not that entangled. 

@ipa-mdl what do you think?